operator/identity/cell.go

```diff
@@ -0,0 +1,82 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package identity
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/spf13/pflag"
+
+	operatorOption "github.com/cilium/cilium/operator/option"
+	"github.com/cilium/cilium/pkg/defaults"
+	"github.com/cilium/cilium/pkg/hive/cell"
+)
+
+// TODO: refactor metrics code to use common helpers
+
+// IdentitiesGCCell is a cell that implements a periodic Cilium identities
+// garbage collector.
+// The GC subscribes to identities events to mark all the related identities
+// as alive. If an identity has no activity for a prolonged interval,
+// it is first marked for deletion and eventually deleted.
+var IdentitiesGCCell = cell.Module(
+	"k8s-identities-gc",
+	"Cilium identities garbage collector",
+
+	cell.Config(defaultIdentityGCConfig),
+
+	cell.Provide(newIdentitiesGC),
+
+	// Invoke forces the instantiation of the identity gc
+	cell.Invoke(func(*identitiesGC) {}),
+)
+
+// IdentitiesGCConfig contains the configuration for the identity-gc.
+type IdentitiesGCConfig struct {
+	IdentityGCInterval       time.Duration
+	IdentityHeartbeatTimeout time.Duration
+
+	IdentityGCRateInterval time.Duration
+	IdentityGCRateLimit    int64
+}
+
+var defaultIdentityGCConfig = IdentitiesGCConfig{
+	IdentityGCInterval:       defaults.KVstoreLeaseTTL,
+	IdentityHeartbeatTimeout: 2 * defaults.KVstoreLeaseTTL,
+
+	IdentityGCRateInterval: time.Minute,
+	IdentityGCRateLimit:    2500,
+}
+
+func (def IdentitiesGCConfig) Flags(flags *pflag.FlagSet) {
+	flags.Duration(operatorOption.IdentityGCInterval, def.IdentityGCInterval, "GC interval for security identities")
```

Comment:
Hmm, if it's easy it might be worth moving `operatorOption.IdentityGCInterval` etc. definitions here. I think it'll end up being confusing soon when more things move out from OperatorConfig but the flag names remain there.